services/src/main/java/org/keycloak/authentication/authenticators/x509/ValidateX509CertificateUsername.java

```diff
@@ -63,7 +65,7 @@ public void authenticate(AuthenticationFlowContext context) {
         if (config == null) {
             logger.warn("[ValidateX509CertificateUsername:authenticate] x509 Client Certificate Authentication configuration is not available.");
             context.getEvent().error(Errors.USER_NOT_FOUND);
```

Comment:
This looks wrong. Config is missing, but it is showing that the user is not found